### DISCUSSION TITLE
Variable heights transform is wrong

### DISCUSSION THREAD

**Discussion by soniktrooth** [clauderic/dnd-kit#682] [Q&A]
I have a vertical list with variable height items. I have set the height on these items (as per the variable height story) and I am using the `verticalListSortingStrategy` however when dragging, the transform seems to offset each item that is being dragged over by it's own height, not the height of the item being dragged. Any vague idea what could be going wrong?

**Comment by clauderic** [comment]
Hard to say without seeing some code. Best way for me to help is if you can set up a CodeSandbox replicating the issue you are facing.

**Comment by voigtboy** [comment]
I'm having the same issue.  I'm providing an example video.  Watch when 'iSCSIs" moves through items with larger heights.  "iSCSIs" is taking on the height of the overlapped item rather than maintaining it's own.  This is just the default implementation of useSortable.

https://github.com/clauderic/dnd-kit/assets/7797288/b0029a5f-c431-4513-b209-776056615170

**Comment by l-portet** [comment]
Answer here https://github.com/clauderic/dnd-kit/issues/117#issuecomment-789863258
